### PR TITLE
new pull request (?)

### PR DESCRIPTION
**Pull request by notyouravgjen**

### CODE REVIEW

File: scripts/people/2014/fall/annstan.yaml [hunk 1]
```diff
@@ -6,9 +6,14 @@ irc: AnnStan
...
-    
+    litreview1: http://ann-stan.blogspot.com/2014/09/lit-review-what-is-open-source.html
+    quiz1: http://ann-stan.blogspot.com/2014/09/this-is-quiz-1.html
+    bugfix: http://ann-stan.blogspot.com/2014/09/bugfix-assignment.html
+    meetup1: http://ann-stan.blogspot.com/2014/09/irl-rocpy-meeting-916.html
+    teamprop1: http://ann-stan.blogspot.com/2014/09/team-proposal.html
+    commarchreport: http://ann-stan.blogspot.com/2014/09/community-architecture.html
+    commarchpreso: https://docs.google.com/presentation/d/1yYUtLupf6jJ1GfbAhk1-S4IdGj1yk4TcSQwbmNNswRI/edit?usp=sharing
+    hackathon1: http://ann-stan.blogspot.com/2014/09/girl-develop-it-launch-party.html
+	curriculum: http://ann-stan.blogspot.com/2014/10/4th-grade-math-curriculum.html
```
Comment (ryansb) [inline scripts/people/2014/fall/annstan.yaml]: This is a tab.

**Comment by ryansb** [comment]
From the build system:

```
annstan.yaml is broken: while scanning for the next token found character '\t' that cannot start any token
in "/home/travis/build/decause/hflossk/scripts/people/2014/fall/annstan.yaml", line 17, column 1
```

Please make sure you use spaces, not tabs for indentation.

**Comment by decause** [comment]
@jas2715 

Come see me at the hacakthon tonight, and we'll sort out your .yaml situation.

**Comment by decause** [comment]
@jas2715 ping.

Let's figure out how to fix this thing.

**Comment by decause** [comment]
@jas2715

:+1:

**Pull request merged**